fonts.css

```diff
@@ -0,0 +1,24 @@
+@font-face {

```

Comment:
Create folders for your css/js files
css
-- index-page.css (rename index.css)
-- catalog.css 
js
-- script.js
and html files without any folder, just leave it in root foler.
and rename html files
nerds-catalog.html -> catalog.html
nerds-index.html -> index.html
1) First of all, your root folder has name 'nerds bla bla', for what have you used nerds prefix? I don't know
2) Basically, the main html file has 'index.html' name.